Makefile

```diff
@@ -75,32 +99,33 @@ docs-user: clean-docs
 docs: docs-user docs-developer
 
 release:
-	ls -l dist/
-	read "\nDo you want to upload everything in dist/*?\n\n CTRL+C to exit."
+	@ls -l dist/
+	@read "\nDo you want to upload everything in dist/*?\n\n CTRL+C to exit."
```

Comment:
When I did `make release`, it only showed something like
```
total 237264
-rw-r--r--  1 lingyiwang  staff  57507242 Jan 24 16:23 kolibri-0.8.0.dev020180124211410-git.tar.gz
-rw-r--r--  1 lingyiwang  staff  63968248 Jan 24 16:23 kolibri-0.8.0.dev020180124211410_git-py2.py3-none-any.whl
```
I think this is because in `read TEXT`, `TEXT` stands for a parameter and stores the user input, so "\nDo you want to upload everything in dist/*?\n\n CTRL+C to exit." is not prompted.
I tried with adding a prompt option `@read -p "\nDo you want to upload everything in dist/*?\n\n CTRL+C to exit."` and then got
```
total 237264
-rw-r--r--  1 lingyiwang  staff  57507242 Jan 24 16:23 kolibri-0.8.0.dev020180124211410-git.tar.gz
-rw-r--r--  1 lingyiwang  staff  63968248 Jan 24 16:23 kolibri-0.8.0.dev020180124211410_git-py2.py3-none-any.whl
\nDo you want to upload everything in dist/*?\n\n CTRL+C to exit.
```
But I still got `\n` in the prompt instead of the real newlines. I'm not sure how to resolve this part of the problem.